py/tests/test_crashprobe.py

```diff
@@ -364,7 +364,7 @@ def test_jump_into_an_nx_page(res_path, make_report_sym, version, build, arch):
 
     # So let's assert for the second best
     else:
-        assert basename(bt[0]["full_path"]) is None
+        assert "full_path" not in bt[0]
```

Comment:
The frame not being mapped causes the `"full_path"` key to not be set on this frame, so the test needs to be fixed.

Reply:
I’m surprised we still have python tests for this. didn’t we want to get rid of the python bindings for this a long time ago? :-D

Reply:
Yeah tbh I was quite surprised when I saw that `symbolic` ships a Python-based mini-Symbolicator for testing purposes